DataHandler.py

```diff
@@ -304,6 +308,18 @@ def parseFiles(self):
 		ins.close()
 		self.loadCertificates()
 
+	def set_latest_spring_version(self):
```

Comment:
IMHO this should be removed: the benefit is very low and it could cause weird behaviour.

Reply:
I thought you might appreciate not having to call SETMINSPRINGVERSION every time the server was restarted. Since you don't want it (and I have no use for it), will remove.

This code is essentially identical to https://github.com/spring/uberserver/blob/master/DataHandler.py#L427. I don't know what is meant by "wierd behaviour".

Reply:
> I don't know what is meant by "wierd behaviour".

undefined / non-deterministic behaviour. using this param could cause the server not to start up if the webserver is not reachable. also the min version set very likely is not the version which should be set when using this param / function.

Reply:
If this could cause the server not to start, the same issue presumably exists with the existing detectIp function, which contacts the same webserver. (I don't think this happens though - when I tested it with my internet disconnected, it just waited for the timeout period set and then started as normal.)